### ISSUE TITLE
No app.php in root folder

### ISSUE THREAD

**Issue by CherryDT** [SolidInvoice/SolidInvoice#1001]
**Describe the bug**
No app.php exists although the nginx config directs all requests to it, so the software can't work.

**To Reproduce**
1. Go to the docs page that describes how to install from distribution package: https://docs.solidinvoice.co/en/latest/guide/getting_started/downloading.html#installing-distribution-package-1
2. Follow the steps there - download latest release (SolidInvoice-2.2.5.zip) from https://github.com/SolidInvoice/SolidInvoice/releases and copy contents into web server directory, set up nginx config as shown on the docs page as well
3. Try to open page
4. Note it doesn't work, also note nginx config refers to /app.php which doesn't exist (in fact I cant find any file named "app.php" anywhere in the extracted files!)

**Expected behavior**
Working installation

**Comment by pierredup** [comment]
The setup docs are outdated and not up to date with the latest version (we are busy completely revamping the docs which is why the old docs haven't been updated yet).

The webroot and front controller needs to be set to `public/index.php` instead of the `web` folder and `app.php`

**Comment by CherryDT** [comment]
OK I will try.

By the way there are more errors in the docs. I tried installing from Git (to get the latest fixes) and `php composer.phar install` fails with `composer.phar` not existing (there is `composer.json`, though).

**Comment by CherryDT** [comment]
I still had issues making it work with nginx, now I wanted to try the PHP dev server, just to get started, but I again got an issue:

```
$ php app/console server:run
Could not open input file: app/console
```

I looked for a `console` file and thought OK maybe I need `bin/console`, but that failed too, saying command `server:run` is not defined.

**Comment by CherryDT** [comment]
I did get it working in nginx now.

Last question, how do I disable the debug bar at the bottom?

**Comment by pierredup** [comment]
You can edit the `.env` file (or `.env.dist` if the `.env` doesn't exist) and set the following values:

```
SOLIDINVOICE_ENV=prod
SOLIDINVOICE_DEBUG=0
```

This will run the app in a production-like mode, without any of the debug utilities (although if you want to make any changes to the source, I would suggest leaving the values as-is, since it helps a lot with debugging).
If you want to run a production instance, then I would recommend to rather use the distributions package (zip/tar.gz) files or the Docker images. Cloning and setting up using Git is mostly only needed for development purposes.

**Comment by CherryDT** [comment]
I see.

I set it up from Git mainly because I really needed the fix for #958 and when I started working on this, it was not available in a release yet (I saw now that in the meantime you launched 2.2.6 which includes the fix). Also, I really need the API support for adding payment and the notes field (both in UI and API). Without those three things I can't use the software for the intended purpose. So I was trying to get those things ready somehow. I was assuming I'd also need to use this sort of setup later in production because when I would make changes to get the payment record stuff done, I'd also have to get those changes into production on my end afterwards as soon as I can.
